core_tools/GUI/keysight_videomaps/plotter/plotting_functions.py

```diff
@@ -293,7 +293,7 @@ def init_plot(self):
             irow = i // n_col
             self.top_layout.addWidget(plot_2D, irow, icol, 1, 1)
 
-            range1 = self.parameter_getter.setpoints[0][1][0][-1]
+            range1 = self.parameter_getter.setpoints[0][1][-1]
```

Comment:
Unfortunately this change is not correct. 
The setpoints of the qcodes multiparameters consists of a tuple per dimension where each tuple is of a higher dimension. In this case setpoints[0] contains the setpoints for channel 0. setpoints[0][0] is a 1D array for the slow sweep. setpoints[0][1] is a 2D array. So setpoints[0][1][0][-1] is the last value of the second dimension at the end of the first slow sweep.